go/kbnm/hostmanifest/whitelist.go

```diff
@@ -0,0 +1,57 @@
+package hostmanifest
+
+import (
+	"encoding/json"
+	"os"
+	"path/filepath"
+)
+
+// whitelistPath is used for installing the whitelist as a JSON into a given
+// path on disk.
+type whitelistPath struct {
+	// Root is the path of the machine-global NativeMessage whitelists.
+	Root string
+	// Home is the path of the NativeMessage whitelists for regular users.
+	Home string
+}
+
+func (w *whitelistPath) path(u User, appID string) string {
+	if u.IsAdmin() {
+		return filepath.Join(u.PrefixPath(), w.Root, appID+".json")
```

Comment:
Maybe it should be `RootNativeMessagingWhitelistLocation` vs `HomeNativeMessagingWhitelistLocation` or something. Given that it's within the context of `hostmanifest` package in the whitelist helpers... can you think of better names?

I'm not married to Root/Home by any means. Global vs UserLocal might be better?